client/src/ehr_components/Visits/VisitInfo.tsx

```diff
@@ -0,0 +1,86 @@
+import { PatientVisit } from "../data/new-visit-fields";
+
+interface VisitInfoProps {
+  className: string;
+  selectedVisit?: PatientVisit;
+  isAnimating: number;
+  setIsAnimating: Function;
+}
+
+let sectionTitles = [
```

Comment:
Could this be an object rather than array of arrays ?

Reply:
I'm not sure about the outer array. I believe that one is needed because the order determines the order of what's displayed on the screen. The inner arrays could certainly be objects though!